### PR TITLE
Add resourcegroup to request

### PR DESCRIPTION
**Pull request by cdamian**

### CODE REVIEW

File: src/Apib.php [hunk 1]
```diff
@@ -78,6 +78,7 @@ private function processApiHttpTransactions(
         $req = new SpasRequest();
 
         if ($apiResourceGroup->getTitle()) {
+            $req->setResourceGroup($apiResourceGroup->getTitle());
```
Comment (hendrikmaus) [inline src/Apib.php]: According to my suggestion in https://github.com/hendrikmaus/spas-parser/pull/2/files#diff-85bbfd9c8354c84ff7bb5d1b61c8ffedR28 we should pull `$req->setResourceGroup($apiResourceGroup->getTitle());` out of the if condition here, change it to `$req->setResourceGroup($apiResourceGroup);` and call it above the `if` condition.

This way, every request would carry a reference to the group it was in.

Comment (cdamian) [inline src/Apib.php]: Took me a while to get it :P

Changes were added accordingly in a new commit.

**Comment by hendrikmaus** [comment]
@cdamian only thing left is to adjust the unit test - it is failing

**Comment by hendrikmaus** [comment]
I updated `hmaus/spas-parser` in master.

A `rm -rf composer.lock vendor/ && composer install` should get you the latest code; IDE should also recognise the new method you created 🍄

**Comment by cdamian** [comment]
@hendrikmaus All green :D

**Pull request merged**